servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -758,10 +896,38 @@ public void serverStreamingCall(final GrpcServiceContext ctx, final CompatReques
                             responseWriter.write(computeResponse(i));
                         }
                     }
-                }));
+                }, codings));
         return TestServerContext.fromServiceTalkServerContext(serverContext);
     }
 
+    @Nonnull
```

Comment:
We do not need to explicitly add `@Nonnull` annotation, it's always added via `@ElementsAreNonnullByDefault`.